### PR TITLE
Updating Getting-Started, moving development instruction into separat…

### PR DESCRIPTION
**Pull request by trisberg**
…e document

### CODE REVIEW

File: Developing-with-Minikube.adoc [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+* A running 1.7+ Kubernetes cluster
+
+TIP: If using Minikube, we recommend using Minikube v0.23.0 or v0.24.1 and avoiding v0.24.0 due to some DNS issues
+
+* Kubernetes CLI `kubectl` installed and on the local system PATH. We recommend using the same version or later as the Kubernetes cluster you are using.
+
+* Docker, you will need Docker installed. We have used Docker version 17.x or later.
+
+* Helm, you need helm installed, see instructions link:Getting-Started.adoc#helm[here].
```
Comment (ericbottard) [inline Developing-with-Minikube.adoc]: This guide is very Helm focused, which IMO doesn't reflect the title "__developing__ with minikube". I think the contents of "manual build and deploy" is more appropriate for this.
This guide is more about "how to tweak a single component of the helm chart here and there", which as a matter of fact may be needed in a production setup.

WDYT?

File: README.adoc [hunk 2]
```diff
@@ -52,9 +52,13 @@ A FaaS for Kubernetes
...
 
-== Installation
+== Installation of latest release
 
-You can install the system using link:Getting-Started.adoc#helm[a Helm chart] or by link:#manual[manually building and deploying] the components.
+You can install the latest release using link:Getting-Started.adoc[a Helm chart]. 
+
+== Developer installation
+
+If you want to develop riff you can install the current development version with the link:Developing-with-Minikube.adoc[Helm chart] or by link:#manual[manually building and deploying] the components.
```
Comment (ericbottard) [inline README.adoc]: Regarding my above comment, this is more to the point.
I guess then, that the only thing I'm not at ease with is the choice of words "Developing riff with minikube". Two giveaways that there is something phony IMO:
- You use "Helm Chart" as the link title
- The linked document isn't that specific about minikube, when you think about it

File: Developing-with-Minikube.adoc [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+== Prerequisites
+
+* A running 1.7+ Kubernetes cluster
+
+TIP: If using Minikube, we recommend using Minikube v0.23.0 or v0.24.1 and avoiding v0.24.0 due to some DNS issues
+
+* Kubernetes CLI `kubectl` installed and on the local system PATH. We recommend using the same version or later as the Kubernetes cluster you are using.
+
+* Docker, you will need Docker installed. We have used Docker version 17.x or later.
```
[the remark this answers is not in the document]
Comment (trisberg) [reply Developing-with-Minikube.adoc]: don't think so, wouldn't most developer want Docker anyway?

File: Developing-with-Minikube.adoc (hunk 1 above)
Comment (trisberg) [reply Developing-with-Minikube.adoc]: Yes, would agree, changing titles and some content to reflect this

File: README.adoc (hunk 2 above)
Comment (trisberg) [reply README.adoc]: Fixing

File: Development-Helm-install.adoc [hunk 4]
```diff
@@ -108,7 +114,7 @@ helm install riffrepo/riff --name control --set functionController.image.tag=0.0
 
 * Overriding the image repository and version tag of a riff component with a custom built component image:
 +
-To set the image repository to `mycustom/function-controller` and the version tag to `0.0.4-test.1` for the `functionController`, use the following:
+To set the image repository to `mycustom/function-controller` and the version tag to `0.0.4-test.1` for the `functionController`, use the something like following:
```
Comment (ericbottard) [inline Development-Helm-install.adoc]: typo, will fix

File: README.adoc [hunk 5]
```diff
@@ -58,7 +58,7 @@ You can install the latest release using link:Getting-Started.adoc[a Helm chart]
 
 == Developer installation
 
-If you want to develop riff you can install the current development version with the link:Developing-with-Minikube.adoc[Helm chart] or by link:#manual[manually building and deploying] the components.
+If you want to run the current development version of riff read the link:Development-Helm-install.adoc[Helm chart] doc. You an also link:#manual[build and deploy riff manually], this is the best option if you want to modify the riff components.
```
Comment (ericbottard) [inline README.adoc]: typo

**Comment by ericbottard** [comment]
Merged as 134012c

**Pull request closed without merging**